src/OpenLoco/CompanyManager.h

```diff
@@ -14,6 +14,17 @@ namespace OpenLoco::CompanyManager
     CompanyId getUpdatingCompanyId();
     void setUpdatingCompanyId(CompanyId id);
 
+    uint8_t getMaxCompetingCompanies();
+    void setMaxCompetingCompanies(uint8_t competingCompanies);
+
+    uint8_t getCompetitorStartDelay();
+    void setCompetitorStartDelay(uint8_t competetorStartDelay);
+
+    uint16_t getMaxLoanSize();
+    void setMaxLoanSize(uint16_t loanSize);
+
+    std::array<CompanyId, 2>& rawPlayerCompanies();
```

Comment:
We already have `getControllingId` and `getSecondaryPlayerId` i dont think we need to export the array.

Reply:
agreed, in a new pr I will remove this